proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -62,6 +68,48 @@ service AccessGraphService {
   // instead of a single response.
   rpc EventsStreamV2(stream EventsStreamV2Request) returns (stream EventsStreamV2Response);
 
+  // AuditLogStream establishes a persistent bidirectional stream for exporting
+  // audit log events from a client (teleport) to a server (access-graph).
+  //
+  // This stream facilitates:
+  // - Initial configuration exchange and validation.
+  // - Streaming batches of audit log events from client to server.
+  // - Reliable export resumption via client-provided resume state updates.
+  // - Server providing the initial resume state to the client upon connection.
+  // - Client requests for server-side bulk export state cleanup.
+  //
+  // Basic Interaction Flow:
+  // 1. Client connects and sends an initial `AuditLogStreamRequest` with `config`.
+  // 2. Server replies with an initial `AuditLogStreamResponse`, confirming the
```

Comment:
the server replies twice. I think it's important to split this into Config response and resume response